docs/EN_US/ECLWatch/ECLWa_mods/SessFaq.xml

```diff
@@ -0,0 +1,255 @@
+
+        <para><emphasis role="strong">A</emphasis>: No. After a configurable
+        period of inactivity your session locks. You then need to unlock to
+        resume your session.</para>
+      </listitem>
+
+      <listitem>
+        <?dbfo keep-together="always"?>
+
+        <para><emphasis role="strong">Q</emphasis>: How long until my password
+        expires?</para>
+
+        <para><emphasis role="strong">A</emphasis>: This depends on your
+        system policies. If your password does expire, you will be presented a
+        dialog to reset it.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Can I log in concurrently
+        with different credentials?</para>
+
+        <para><emphasis role="strong">A</emphasis>: Yes, using different tabs
+        in a single browser, multiple instances of the same browser, or
+        multiple instances of different browsers.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Will I be able to log in
+        as a different user?</para>
+
+        <para><emphasis role="strong">A</emphasis>: Yes, with our new login
+        screen, you can input previously used IDs or enter a different one.
+        You can have as many user sessions active at any time as permitted by
+        your system's resources.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Is there an option to stay
+        logged in forever?</para>
+
+        <para><emphasis role="strong">A</emphasis>: No.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Is there any option to not
+        time out from inactivity?</para>
+
+        <para><emphasis role="strong">A</emphasis>: No.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Will I lose data if I get
+        automatically logged out?</para>
+
+        <para><emphasis role="strong">A</emphasis>: No. You do not get logged
+        out. Your session will get locked. Anything typed into any fields
+        (such as a search box) that has not been submitted or entered could
+        potentially be lost. However, since the session is only locked, it is
+        unlikely that any data will be lost.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Will my queued and
+        scheduled workunits run when I am locked out?</para>
+
+        <para><emphasis role="strong">A</emphasis>: Yes, the session only
+        applies to ESP/ECL IDE and ESP/ECLWatch communications.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Will HPCC command line
+        utilities be affected?</para>
+
+        <para><emphasis role="strong">A</emphasis>: No. They function exactly
+        the same as before session management.</para>
+      </listitem>
```

Comment:
This is only true if ESP is not configured for "AuthTypeSessionOnly."  Confirm with Kevin

Reply:
Still wrong